src/main/kotlin/com/atlassian/performance/tools/awsinfrastructure/api/InfrastructureFormula.kt

```diff
@@ -31,8 +31,16 @@ class InfrastructureFormula<out T : VirtualUsers>(
     private val investment: Investment,
     private val jiraFormula: JiraFormula,
     private val virtualUsersFormula: VirtualUsersFormula<T>,
-    private val aws: Aws
+    private val aws: Aws,
+    private var preProvisionedNetwork: Network?
```

Comment:
Let's use `val` (Java `final`) by default. Mutable references should be the exception.